test/unit/orchestration/dhcp_test.rb

```diff
@@ -154,6 +154,9 @@ def test_bmc_should_have_valid_dhcp_record
     bmc = h.interfaces.bmc.first
     assert !bmc.new_record?
     bmc.mac = bmc.mac.succ
+    binding.pry
```

Comment:
it didn't stop the [test](http://ci.theforeman.org/job/test_develop_pr_core/3664/database=postgresql,ruby=1.9.3/testReport/junit/(root)/DhcpOrchestrationTest/test_0010_when_an_existing_host_change_multiple_attributes__both_his_dhcp_and_bmc_dhcp_records_should_be_updated/) though, anyway do you check everything I do? I don't complain, it just surprise me that I can't hide any mistake :-)